src/ofp_init.c

```diff
@@ -108,6 +108,19 @@ odp_bool_t *ofp_get_processing_state(void)
 	return &shm->is_running;
 }
 
+int ofp_get_global_parameters(ofp_global_param_t *params)
+{
+	if (!params)
+		return -1;
+
+	if (ofp_global_config_lookup_shared_memory() == -1)
+		return -1;
+
+	memcpy(params, &shm->global_param, sizeof(ofp_global_param_t));
```

Comment:
Struct assignment would work as well and would have slightly better type safety.